src/03-deals/quotations.apib

```diff
@@ -156,4 +156,41 @@ Download a quotation in a specific format.
     + Attributes (object)
         + data (object)
             + type: `quotation` (string)
-            + id: `e4bf74ca-e900-471f-84b1-276e5d3afae4` (string)
\ No newline at end of file
+            + id: `e4bf74ca-e900-471f-84b1-276e5d3afae4` (string)
+
+### quotations.update [POST /quotations.update]
+
++ Request (application/json)
+
+    + Attributes (object)
+        + quotation_id: `5b16f6ee-e302-0079-901b-50c26c4a55b1` (string, required)
+        + `grouped_lines` (array, optional)
```

Comment:
I'd keep the remark here that either grouped lines or text is required for a quotation.
```suggestion
        + `grouped_lines` (array, optional) - A quotation needs `grouped_lines` and/or `text` to be valid
```

Reply:
I suppose you can update the discounts without touching the line items or text so I don't think so?

Reply:
Indeed that should be possible. So the question is: does this remark refer to the (updated) quotation or only to the provided data? I assume it refers to the quotation.

I'm OK with removing it provided we also clear it here: https://github.com/teamleadercrm/api/pull/547/files#diff-2625016b50d68d922257f74801cac29cR2240

Reply:
I'll put it back if you think it's clearer that way